Sources/ViewControllers/Products/Search/ScannerViewController.swift

```diff
@@ -63,13 +63,17 @@ class ScannerViewController: UIViewController, DataManagerClient {
         lastCodeScanned = nil
         allergenAlertShown = false
 
-        checkCameraPermissions()
-        configureVideoView()
-        configureSession()
-        configureOverlay()
-        configureFlashView()
-        configureTapToFocus()
-
+        if !CommandLine.arguments.contains("debug") {
```

Comment:
"debug" should not be use, as it implies that the app is run with the debugger attached (which occurs for example automatically when running the project in dev mode). As shown in https://docs.fastlane.tools/getting-started/ios/screenshots/#advanced-snapshot, you can use
```if NSUserDefaults.standardUserDefaults().boolForKey("FASTLANE_SNAPSHOT") {
    // runtime check that we are in snapshot mode
}``` 
to check if you are inside a screenshot test

Reply:
Thanks @philippeauriach ; the issue should be fixed as part of 0ea7c07.

Reply:
One note : we might need to also have this custom behavior (for overriding the scanner, for example) in other UI tests in the future. When we’ll be there, we could add a helper to determine whether or not we are in a test environment.